frontend/clientApi/lens/createProfile.js

```diff
@@ -1,13 +1,12 @@
 import { gql } from '@apollo/client';
 
-export const createProfile = gql`
+export const createProfile = handle => gql`
     mutation CreateProfile {
```

Comment:
Instead of passing handle as a param above you can pass it as part of the mutation...
```suggestion
export const createProfile = gql`
    mutation CreateProfile($handle: Handle!) {
```

Then when you use the mutation query..
```
                const createLensProfile = await lensClient.mutate({
                    mutation: createProfile,
                    variables : { handle: ... }
                });
```